dotnet-server-sdk-shared-tests/LaunchDarkly.ServerSdk.SharedTests.Tests/LaunchDarkly.ServerSdk.SharedTests.Tests.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.1</TargetFrameworks>

```

Comment:
Please don't do this. I know .NET Core 2.0 is EOL, but what you've changed here is not part of this project— it's shared code that's imported as a git subtree, as you'll see in https://github.com/launchdarkly/dotnet-server-sdk-dynamodb/blob/master/dotnet-server-sdk-shared-tests/README.md. Also, by doing this you broke the build, because you didn't update the CircleCI configuration to use a Docker image that has .NET Core 2.1.